### ISSUE TITLE
A tool to move micro parts around without having to wrench them off

### ISSUE THREAD

**Issue by ChromaPIE** [AE2-UEL/Applied-Energistics-2#504]
![图片](https://github.com/user-attachments/assets/504a7885-4e85-4b1b-86ae-c5cb95f1f1d5)

Here you can see a P2P tunnel attached to a cable, let's say it's online and above it is an alloy smelter.

And now you are reconstructing your automation and you have to move the alloy smelter to the left side of that cable, but for some reason, like having a big base or a complex network, you don't want to wrench it, place it and tune the freq all over again and again.

Yeah, we got Carry On, but it only allows for picking up the whole block space, enough for moving the cable and what's attached to it to another location but the facing remains unchanged.

So I'll suggest a new tool akin to the Cardboard Box from Mekanism, or the famous Chest Transporter, but only works for AE2 microparts, right click on any cable/micropart to pick it up, and on a new location to put it down, and it too has the white selection outline for precise placement. All data in the micropart, like its custom name, its freq, or upgrades/patterns/item and fluid settings in interfaces, will be kept.

**Comment by Forever-178** [comment]
I think memory card might be helpful in this case. It can even replicate pattern information from me interface. But it can't copy the information of upgrades.

**Comment by NotMyWing** [comment]
Yeah, that's a desirable addition. I think adding a mode to Network Tool to preserve configs/inventories would be the best of two worlds.
